sources/packages/backend/libs/integrations/src/esdc-integration/e-cert-integration/e-cert-file-handler.ts

```diff
@@ -207,10 +188,7 @@ export abstract class ECertFileHandler extends ESDCFileHandler {
         uploadedRecords: disbursementRecords.length,
       };
     } catch (error: unknown) {
-      if (
-        error instanceof CustomNamedError &&
-        error.name !== ECERT_GENERATION_NO_RECORDS_AVAILABLE
-      ) {
+      if (error instanceof CustomNamedError) {
```

Comment:
Please  move the log.error outside if condition and then remove the if condition.